src/main/java/frc/robot/RobotContainer.java

```diff
@@ -127,7 +132,7 @@ private void configureBindings() {
                 xButton.toggleOnTrue(new MoveArm(m_arm, ARM_DOWN, ARM_DOWN_SPEED).withTimeout(1.1));
                 leftBumper.whileTrue(new MoveWrist(m_wrist, WRIST_IN, MOTOR_SPEED));
                 rightBumper.whileTrue(new MoveWrist(m_wrist, WRIST_OUT, MOTOR_SPEED));
-                leftTigger.whileTrue(new MoveArm(m_arm, ARM_UP, ARM_UP_SPEED));
+                //leftTigger.whileTrue(new MoveArm(m_arm, ARM_UP, ARM_UP_SPEED));
```

Comment:
why did we remove this button.  does Gio only want to control the arm in one direction?

Reply:
See my comment, this file should be reverted - that was for testing